From 4d7825083d185dd36bdb02b29ae5fcbea7e22fb8 Mon Sep 17 00:00:00 2001
From: Daniel Ammann <daniel.ammann@bytesatwork.ch>
Date: Wed, 20 Nov 2019 08:19:03 +0100
Subject: [PATCH] ARM: bytepanel_defconfig: Enable dynamic printk support

It is very helpful, comes with little overhead, and this is a development
kernel.
---
 arch/arm/configs/bytepanel_defconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/configs/bytepanel_defconfig b/arch/arm/configs/bytepanel_defconfig
index 285fcfe8b53e..d0bd1445aabe 100644
--- a/arch/arm/configs/bytepanel_defconfig
+++ b/arch/arm/configs/bytepanel_defconfig
@@ -3845,7 +3845,7 @@ CONFIG_CONSOLE_LOGLEVEL_DEFAULT=7
 CONFIG_CONSOLE_LOGLEVEL_QUIET=4
 CONFIG_MESSAGE_LOGLEVEL_DEFAULT=4
 # CONFIG_BOOT_PRINTK_DELAY is not set
-# CONFIG_DYNAMIC_DEBUG is not set
+CONFIG_DYNAMIC_DEBUG=y
 
 #
 # Compile-time checks and compiler options
-- 
2.20.1

